README.md

```diff
@@ -2,44 +2,145 @@
 
 A library for composable forecasting models built on top of scikit-learn
 
+
+<!-- Add intro that provides project context. -->
+<!-- FTR - final technical report. Everything must link to there - except for this doc. Link to task numbers -->
+<!-- TODO: Get permalink for FTR once published and link it from here! -->
+
+<!-- Explain what this does for lawyers... it solves this particular problem for the FTR - as well as this class of problem in general -->
+<!-- Expand on connections to SOPO: be clear about mapping to the things we completed. Make sure keyword searches will work -->
+<!-- Read SOPO: look for nouns in the relevant task descriptions. -->
+
+
 The API for fit, predict, and metrics is reduced to specifying a start and end times for a given location.
 The model must construct feature data using column transforms. Having done so, forecasting as a service become trivial.
 
 ## Installation
 
-requires the eccodes library: `apt-get install libeccodes-dev`
+<!-- Talk about env for this to run - should be a complete description - provide the context. -->
 
+This library is designed for use by technical engineers and data scientists. It takes advantage of the python
+data science ecosystem and therefore requires installation of many third party open source libraries. It has 
+been developed and tested in a linux operating system. Running on a docker container such as 
+the [canonical Ubuntu image](https://hub.docker.com/_/ubuntu) is strongly recommended. The library was 
+developed using Ubuntu 22.04 (Jammy) with Python 3.10.6.
+
+### Installing system libraries
+Reading the grib weather data requires the eccodes library: `apt-get install libeccodes-dev`
+
+### Installing python libraries
 `pip3 install -r requirements.txt`
 
 Verify you can run the unit tests `python -m unittest`
 
-If running in a container, to start a notebook use `jupyter notebook --NotebookApp.ip=0.0.0.0`
+To start jupyter notebook run `jupyter notebook --NotebookApp.ip=0.0.0.0`
+Now you can open the example notebook and execute the cells in the demonstration.
```

Comment:
````suggestion
To start jupyter notebook run:

```sh
jupyter notebook --NotebookApp.ip=0.0.0.0
```

This will print a URL, which you can open in your browser. Then open the example notebook and execute the cells in the demonstration to get acquainted with the functionality.
````

P3